packages/actor-rdf-resolve-hypermedia-qpf/components/Actor/RdfResolveHypermedia/Qpf.jsonld

```diff
@@ -0,0 +1,68 @@
+{
+  "@context": [
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-rdf-resolve-hypermedia-qpf/^1.0.0/components/context.jsonld",
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/bus-rdf-resolve-hypermedia/^1.0.0/components/context.jsonld"
+  ],
+  "@id": "npmd:@comunica/actor-rdf-resolve-hypermedia-qpf",
+  "components": [
+    {
+      "@id": "carrhq:Actor/RdfResolveHypermedia/Qpf",
+      "@type": "Class",
+      "extends": "cbrrh:Actor/RdfResolveHypermedia",
+      "requireElement": "ActorRdfResolveHypermediaQpf",
+      "comment": "A comunica QPF RDF Resolve Quad Pattern Actor.",
+      "parameters": [
+        {
+          "@id": "carrqph:Actor/RdfResolveQuadPattern/Hypermedia/subjectUri",
```

Comment:
The `@id` of these parameters should have the prefix `carrhq:Actor/RdfResolveHypermedia/Qpf/` (because of the actor that defines these params)